src/main/java/mil/dds/anet/utils/AuthUtils.java

```diff
@@ -50,6 +50,13 @@ public static boolean canAdministrateOrg(final Person user, final String organiz
     if (position.getType() != PositionType.SUPERUSER) {
       return false;
     }
+    // SuperuserType.CAN_CREATE_OR_EDIT_ANY_ORGANIZATION can deal with any organization
+    if (position.getType() == PositionType.SUPERUSER && position
```

Comment:
We already know that `position.getType() == PositionType.SUPERUSER && position` is true from the `if` directly above.